stripe/src/main/java/com/stripe/android/cards/Bin.kt

```diff
@@ -1,8 +1,14 @@
 package com.stripe.android.cards
 
+import com.stripe.android.model.StripeModel
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
 internal data class Bin internal constructor(
     internal val value: String
-) {
+) : StripeModel {
+    override fun toString() = value
```

Comment:
TODO: I think we can change this to an `inline` class and remove this override when `inline` is released.

Reply:
Never mind. I just tested this and my assumption was incorrect.